web/src/types/index.ts

```diff
@@ -52,6 +52,7 @@ export interface Asset {
   created: string,
   modified: string,
   version: Version,
+  zarr?: string,
```

Comment:
Shouldn't we also have the `blob` field here?
```suggestion
  zarr?: string,
  blob?: string,
```

Reply:
Yes. However, there seems to be a lot of issues with the Asset TS types, so I figured I'd make a follow up PR to this one that cleans them up and ensures that they match up with the DRF serializers. In fact, that `zarr` field isn't even needed for this PR since we're using s3 urls now, so I'll drop that from this PR to keep things consistent. 

I might look into something like https://github.com/adenh93/django-typomatic to help us keep our TypeScript interfaces in sync with our DRF serializers.